### DISCUSSION TITLE
How to remove top menu on sidebar

### DISCUSSION THREAD

**Discussion by BPUPH** [piroor/treestyletab#3464] [Q&A]
I am wondering if there is a way to remove this top menu from the sidebar:

![firefox_h7Kx8YcXJE](https://github.com/piroor/treestyletab/assets/161071764/56bcd1ca-63cb-4ba7-b860-423cb38dcc4b)

It gets in the way when it comes to dragging tabs onto other tabs. I don't remember it being there in a previous version I used. I just had to reinstall Firefox and I can't find any option to disable it.

**Comment by irvinm** [comment]
userChrome.css:

`#sidebar-header { display: none; }`

I think this should work, but I did not try.  (This is a Firefox\Sidebar change ... nothing to do with TST)
